README.rdoc

```diff
@@ -19,22 +19,36 @@ Ver. 1.0.x is not 100% backward compatible with 0.3.x. Some methods have been re
 
 First, follow "Create a client ID and client secret" in {this page}[https://developers.google.com/drive/web/auth/web-server] to get a client ID and client secret for OAuth. Set "Application type" to "Other" in the form to create a client ID if you use GoogleDrive.saved_session method as in the example below.
 
+
+The client ID and client secret you obtained in the step above.
+You can keep them in a config file, it should look like this one below:
+
+  {
+    "client_id": "xxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.apps.googleusercontent.com",
+    "client_secret": "xxxxxxxxxxxxxxxxxxxxxxxx",
+    "scope": [
```

Comment:
Can you remove "scope" and "refresh_token" from this example? They can be omitted in most cases. You may put a full example in the RDoc of saved_session method, but the example in README should be minimal.